packages/core/js-client/src/jsPeer/FluencePeer.ts

```diff
@@ -424,7 +424,7 @@ export abstract class FluencePeer {
   ): Promise<CallServiceResult> {
     const particleId = req.particleContext.particleId;
 
-    log_particle.trace(
+    log_particle.debug(
       "id %s. executing call service handler %j",
```

Comment:
maybe show here `serviceId` and `functionName`? 

`req` seems to have way too much info